Sources/ProjectDescription/Config.swift

```diff
@@ -14,6 +14,28 @@ public struct Config: Codable, Equatable {
             case targets([TargetReference])
         }
 
+        /// Contains options for autogenerated targets
+        public enum AutogenerationOptions: Codable, Equatable {
```

Comment:
Unfortunately, we are still doing releases with Xcode 12.5.1 and this won't compile (automatic synthesization of `Codable` conformance was added in Swift 5.5 and Xcode 12.5.1 has Swift 5.4 bundled)

Reply:
Ah I missed that. I don't have Xcode 12.x installed but have added a manual conformance